packages/paypal-js/src/utils.ts

```diff
@@ -62,24 +62,15 @@ export function insertScriptElement({
     document.head.insertBefore(newScript, document.head.firstElementChild);
 }
 
-export function processOptions(options: PayPalScriptOptions): {
+export function processOptions({
+    sdkBaseUrl: customSdkBaseUrl,
+    environment,
+    ...options
+}: PayPalScriptOptions): {
     url: string;
     attributes: StringMap;
 } {
-    const { environment } = options;
-    // Keeping production as default to maintain backward compatibility.
-    // In the future this logic needs to be changed to use sandbox domain as default instead of production.
-    let sdkBaseUrl =
-        environment === "sandbox"
-            ? "https://www.sandbox.paypal.com/sdk/js"
-            : "https://www.paypal.com/sdk/js";
-    // env is not an allowed key for sdk/js url.
-    delete options.environment;
```

Comment:
do we still need to keep this line though? if `environment` isn't an allowed param for sdk/js URL.